localisation/00_County_names.csv

```diff
@@ -2547,6 +2573,10 @@ c_van_zhunod;Van Zhu'nod;Van Zhu'nod;Van Zhu'nod;;Van Zhu'nod;;;;;;;;;x
 c_van_zhunod_adj;Van Zhu'nodian;;Van Zhu'noder;;;;;;;;;;;x
 c_vankeiki;Van'keiki;Van'keiki;Van'keiki;;Van'keiki;;;;;;;;;x
 c_vankeiki_adj;Van'keikian;;van'keiker;;;;;;;;;;;x
+c_vashjir_atol;Vashj'ir Atol;Vashj'ir Atol;Vashj'ir Atol;;Vashj'ir Atol;;;;;;;;;x
+c_vashjir_atol_adj;Vashj'ir Atolian;;Vashj'ir Atoler;;;;;;;;;;;x
```

Comment:
Fixed spelling. Atollian and Atoller isn't great English, but I can't think of anything better, so I left it as is.
```suggestion
c_vashjir_atoll_adj;Vashj'ir Atollian;;Vashj'ir Atoller;;;;;;;;;;;x
```